frontend/src/containers/Auth/Login.tsx

```diff
@@ -4,8 +4,8 @@ import Logo from "../../assets/quiz_qrafter_logo_dark.svg"; // Use a regular ima
 import { useAuth } from "../../services/auth";
 import styles from "./auth.module.css";
 
-const SignIn = () => {
-  const { authenticated, signIn } = useAuth();
+const Login = () => {
+  const { authenticated, Login } = useAuth();
```

Comment:
Seems you missed here and [line 61](https://github.com/QuizQrafter/quiz-qrafter/pull/13/files#diff-b0259f43843454dea32956dc5727215037ef77b8ab9a768a8e3d442916360794R61) when renaming `Login` to `login`.

Reply:
My bad, I've made a new commit to resolve this issue.